src/SprykerEco/Zed/Heidelpay/Business/Adapter/Basket/Basket.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Business\Adapter\Basket;
+
+use Generated\Shared\Transfer\HeidelpayBasketRequestTransfer;
+use Generated\Shared\Transfer\HeidelpayBasketResponseTransfer;
+use Heidelpay\PhpBasketApi\Object\Authentication;
+use Heidelpay\PhpBasketApi\Object\Basket as HeidelpayBasket;
+use Heidelpay\PhpBasketApi\Request;
+use SprykerEco\Zed\Heidelpay\Business\Adapter\Mapper\BasketRequestToHeidelpayInterface;
+use SprykerEco\Zed\Heidelpay\Business\Adapter\Mapper\BasketResponseFromHeidelpayInterface;
+use SprykerEco\Zed\Heidelpay\HeidelpayConfigInterface;
+
+class Basket implements BasketInterface
```

Comment:
Please align names `BasketAdapter` and `BasketAdapterInterface`

Reply:
Follow module structure.